packages/api-explorer/src/components/Header/svg/index.tsx

```diff
@@ -0,0 +1,43 @@
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 74 20">
+      <g clipPath="url(#clip0)" fill="#5F6368">
+        <path d="M6.056 0A1.719 1.719 0 004.63 2.678l.734-.734a.725.725 0 11.463.462l-.733.733A1.718 1.718 0 106.056 0zM5.373 5.006a2.67 2.67 0 00-.535-1.609l-.953.952a1.366 1.366 0 01-.261 1.648l.518 1.265a2.685 2.685 0 001.23-2.256z" />
+        <path d="M2.71 6.372h-.024a1.367 1.367 0 11.75-2.509l.944-.943a2.686 2.686 0 10-1.152 4.718l-.517-1.266zM6.092 7.466c-.596 0-1.189.086-1.76.258l.755 1.844a4.112 4.112 0 11-.924.357l-.746-1.839a6.095 6.095 0 102.68-.62h-.005zM18.09 2.52h1.714v12.935h6.309v1.629H18.09V2.52zM28.597 16.383a4.91 4.91 0 01-1.829-1.916 5.533 5.533 0 01-.655-2.68 5.533 5.533 0 01.655-2.676 4.91 4.91 0 011.829-1.916 5.298 5.298 0 015.28 0 4.925 4.925 0 011.83 1.916c.44.824.666 1.746.655 2.68a5.544 5.544 0 01-.655 2.68 4.924 4.924 0 01-1.829 1.916 5.298 5.298 0 01-5.28 0l-.001-.004zm4.348-1.261a3.395 3.395 0 001.275-1.32c.33-.62.496-1.313.48-2.015a4.074 4.074 0 00-.48-2.015 3.395 3.395 0 00-1.272-1.32 3.352 3.352 0 00-1.711-.46 3.4 3.4 0 00-3.003 1.78 4.083 4.083 0 00-.478 2.015 4.081 4.081 0 00.478 2.015 3.417 3.417 0 004.714 1.32h-.003zM39.735 16.383a4.926 4.926 0 01-1.828-1.916 5.544 5.544 0 01-.656-2.68 5.545 5.545 0 01.656-2.676 4.925 4.925 0 011.828-1.916 5.298 5.298 0 015.28 0 4.917 4.917 0 011.83 1.916c.441.824.666 1.746.656 2.68a5.543 5.543 0 01-.656 2.68 4.916 4.916 0 01-1.828 1.916 5.297 5.297 0 01-5.28 0l-.002-.004zm4.349-1.261a3.404 3.404 0 001.271-1.32c.33-.62.496-1.313.48-2.015a4.085 4.085 0 00-.48-2.015 3.403 3.403 0 00-1.271-1.32 3.352 3.352 0 00-1.712-.46 3.408 3.408 0 00-1.721.46c-.541.313-.985.77-1.281 1.32a4.073 4.073 0 00-.48 2.015 4.07 4.07 0 00.48 2.015c.296.55.74 1.007 1.28 1.32a3.409 3.409 0 001.722.46 3.352 3.352 0 001.716-.46h-.004zM48.731 2.52h1.713v9.207L55.6 6.489h2.175v.08l-4.35 4.391 4.153 6.047v.077h-2.054l-3.306-4.939-1.774 1.778v3.16h-1.713V2.52z" />
+        <path d="M59.788 16.402a4.75 4.75 0 01-1.77-1.887 5.707 5.707 0 01-.635-2.712 5.95 5.95 0 01.599-2.647 4.791 4.791 0 011.7-1.947 4.529 4.529 0 012.534-.724c.978 0 1.825.218 2.543.655a4.359 4.359 0 011.657 1.81c.396.825.593 1.731.573 2.646 0 .17-.013.34-.039.509h-7.905c.04.759.222 1.398.548 1.917.3.497.73.902 1.242 1.173a3.372 3.372 0 001.575.392c1.303 0 2.287-.6 2.953-1.8l1.408.684a4.96 4.96 0 01-1.74 1.908c-.744.47-1.637.705-2.68.705-.901.015-1.788-.221-2.563-.681zm5.441-5.671a3.244 3.244 0 00-.352-1.252 2.7 2.7 0 00-.978-1.056c-.443-.287-1.004-.431-1.682-.431a2.85 2.85 0 00-1.995.753c-.547.502-.906 1.164-1.076 1.986h6.083zM68.095 6.49h1.632v1.968h.08c.202-.565.592-1.035 1.17-1.411a3.278 3.278 0 011.81-.561c.415-.01.827.065 1.21.222v1.834a3.21 3.21 0 00-1.45-.323 2.389 2.389 0 00-1.37.424c-.423.29-.767.682-.999 1.14a3.305 3.305 0 00-.372 1.545v5.756h-1.71V6.49z" />
+      </g>
+      <defs>
+        <clipPath id="clip0">
+          <path fill="#fff" d="M0 0h74v19.652H0z" />
+        </clipPath>
+      </defs>
+    </svg>
```

Comment:
Is this the Looker logo? If so it should be available via @looker/components i think?

Reply:
It is, yeah. Let me take a look at that and see if I can update it.

Reply:
After speaking with Jared the Icon version of our logo is not ideal for this use case. If we have a Logo component at some point we can update.